code/modules/unit_tests/armor_verification.dm

```diff
@@ -0,0 +1,33 @@
+/// Verifies that armor procs are working as expected
+/datum/unit_test/armor_verification
+
+/datum/unit_test/armor_verification/Run()
+	var/obj/dummy = allocate(/obj)
+
+	UNLINT(dummy.armor_type = /datum/armor/none)
```

Comment:
Instead of doing this, make a subtype where the armor type is none

Reply:
I forgot I have a dedicated proc for stuff like this

Reply:
```suggestion
	dummy.set_armor(/datum/armor/none)
```